packages/web-components/src/components/card/__stories__/card.stories.react.tsx

```diff
@@ -187,6 +189,83 @@ Static.story = {
   },
 };
 
+Static.story = {
+  parameters: {
+    ...readme.parameters,
+    knobs: {
+      Card: ({ groupId }) => {
+        const image = boolean('Add image:', false, groupId);
+        const eyebrow = textNullable('Eyebrow:', 'SPSS Statistics', groupId);
+        const heading = textNullable('Heading:', 'Free trial', groupId);
+        const copy = textNullable(
+          'Body copy:',
+          'Enjoy full SPSS Statistics capabilities including all add-ons. ' +
+            'All trial registrants are restricted to one free trial per computer per user.',
+          groupId
+        );
+        const tagGroup = boolean('Add tags:', false, groupId);
+        const cta = boolean('Add CTA:', false, groupId);
+        const ctaCopy = cta ? textNullable('CTA copy:', 'Sign up for the trial', groupId) : '';
+        const outlinedCard = boolean('Outlined card:', true, groupId);
+        return {
+          alt: 'Image alt text',
+          defaultSrc: imgXlg4x3,
+          image,
+          eyebrow,
+          heading,
+          copy,
+          tagGroup,
+          cta,
+          ctaCopy,
+          outlinedCard,
+        };
+      },
+    },
+  },
+};
```

Comment:
looks like this is a duplicate from above and might be causing the static [deploy preview](https://ibmdotcom-web-components-react.s3-web.us-east.cloud-object-storage.appdomain.cloud/deploy-previews/7786/index.html?path=/story/components-card--static) for react wrapper to look like this: 
<img width="1087" alt="Screen Shot 2021-12-02 at 10 32 06 AM" src="https://user-images.githubusercontent.com/20210594/144482141-b51b689e-113a-4223-8aea-bfc8bc6be16a.png">